### PR TITLE
fix(package.json): update dependencies to fix vulnerabilities

### PR DESCRIPTION
**Pull request by apaparazzi0329**

### CODE REVIEW

**Comment by germanattanasio** [comment]
@apaparazzi0329 We are still using webpack 4 in this PR. Can we get all the dependencies updated including major releases?

**Comment by apaparazzi0329** [comment]
Sure

**Comment by apaparazzi0329** [comment]
@germanattanasio Does it look ready to go?

**Comment by germanattanasio** [comment]
Did you test this with the node-sdk and make sure it works in the different browsers? I found that this package it's tricky and there are always issues when testing with firefox or chrome.

**Comment by germanattanasio** [comment]
Once you confirm that you manually tested this PR then we can merge it

**Comment by apaparazzi0329** [comment]
The new commit I pushed passed manual testing for chrome, firefox, and safari

File: package.json [hunk 1]
```diff
@@ -38,58 +38,58 @@
...
     "buffer-from": "^1.1.1",
-    "camelcase": "^5.3.1",
+    "camelcase": "^6.2.0",
     "clone": "^2.1.2",
-    "core-js": "^3.6.5",
+    "core-js": "^3.8.3",
     "defaults": "^1.0.3",
     "get-user-media-promise": "^1.1.4",
-    "ibm-watson": "^5.6.0",
+    "ibm-watson": "^6.0.2",
```
Comment (germanattanasio) [inline package.json:82]: Should we make this a feature release since you are doing a major release of the ibm-watson sdk?

Comment (apaparazzi0329) [reply package.json:84]: There is a separate issue [here](https://github.ibm.com/Watson/developer-experience/issues/9005) asking for a proper 1.0 release of this sdk; however, upgrading to `ibm-watson 6.0.2` doesn't actually release any new features to this sdk so I don't think the upgrade itself would justify a feature release

**Review by dpopp07 — APPROVED** [review]
Looks good to me 👍

File: package.json (hunk 1 above)
Comment (germanattanasio) [reply package.json:84]: didn't we drop support for Node <10 in ibm-watson 6.x?

Comment (apaparazzi0329) [reply package.json:84]: We did and I can update the package.json to reflect that but I don't see what new features that would give the user in terms of using the sdk and its methods

Comment (germanattanasio) [reply package.json:84]: Nevermind Angelo

**Pull request merged**